changelog/unreleased/set-mountpoint-on-autoaccept.md

```diff
@@ -0,0 +1,5 @@
+Bugfix: Set the mountpoint on auto accept
+
+On shares auto accept set a mountpoint with same logic as ocs handler
+
+https://github.com/owncloud/ocis/pull/7460
```

Comment:
Nit: Would be cool to have the fixed issues linked here and in the commit message as well .

Reply:
What do you mean by "in the commit message as well"? How should the commit message look like?